server/src/http/broadcaster.rs

```diff
@@ -0,0 +1,190 @@
+/// copied from https://github.com/actix/examples/blob/master/server-sent-events/src/broadcast.rs
+use std::{
+    collections::HashMap,
+    hash::{Hash, Hasher},
+    sync::Arc,
+    time::Duration,
+};
+
+use actix_web::{
+    rt::time::interval,
+    web::{Json, Query},
+};
+use actix_web_lab::{
+    sse::{self, Event, Sse},
+    util::InfallibleStream,
+};
+use dashmap::DashMap;
+use futures_util::future;
+use parking_lot::Mutex;
+use serde::Serialize;
+use tokio::sync::mpsc;
+use tokio_stream::wrappers::ReceiverStream;
+use unleash_types::client_features::{ClientFeatures, Query as FlagQuery};
+
+use crate::{
+    filters::{filter_client_features, name_prefix_filter, project_filter, FeatureFilterSet},
+    tokens::cache_key,
+    types::{EdgeToken, FeatureFilters},
+};
+
+#[derive(Debug, Clone, PartialEq, Eq, Serialize)]
+struct QueryWrapper {
+    query: FlagQuery,
```

Comment:
check if the hashes feature is used for unleash-types. It should be hashable.

(update unleash types to impl hash)

Reply:
I'm guessing we decided to leave this one for later?

Reply:
Handling in https://github.com/Unleash/unleash-types-rs/pull/43